sequencer/src/api/options.rs

```diff
@@ -79,7 +82,7 @@ impl Options {
     pub async fn serve<N, F>(mut self, init_handle: F) -> anyhow::Result<SequencerNode<N>>
     where
         N: network::Type,
-        F: FnOnce(Box<dyn Metrics>) -> BoxFuture<'static, (Consensus<N>, NodeIndex)>,
+        F: FnOnce(Box<dyn Metrics>) -> BoxFuture<'static, (Consensus<N>, NodeIndex, StateKeyPair)>,
```

Comment:
Getting to the point where we might consider replacing the return type tuple with a struct

Reply:
I changed this to `SequencerContext`. [f7033f7](https://github.com/EspressoSystems/espresso-sequencer/pull/879/commits/f7033f7222880be8e53eeee798c12ca2691f496c)